### PR TITLE
Update 2

### PR DESCRIPTION
**Pull request by bo3mrh**
- Mainly modifying PHEHX.py and CoaxialHX.
- Correcting typos.
- Adjust some formulas to reduce computation.
- Editorial comment changes.
- Additional comment for more clarification.

### CODE REVIEW

File: ACHP/CoaxialHX.py [hunk 1]
```diff
@@ -242,15 +247,15 @@ def _TwoPhase_Forward(self,w_2phase=1.0,xout_2phase=1.0):
         pdew_cond=PropsSI('P','T',Tdew_cond,'Q',1.0,'R290')
         h=PropsSI('H','T',Tdew_cond-7,'P',pdew_cond,'R290') #*1000
         params={
-                'ID_i':0.0278,
-                'OD_i':0.03415,
-                'ID_o':0.045,
+                'ID_i':0.0278,      #inner tube, Internal Diameter (ID)
+                'OD_i':0.03415,     #inner tube, Outter Diameter (OD)
```
Comment (ibell) [inline ACHP/CoaxialHX.py]: Outer not Outter

File: Documentation/Condenser.aux [hunk 2]
```diff
@@ -1,5 +0,0 @@
-\relax
```
Comment (ibell) [inline Documentation/Condenser.aux]: Don't commit .aux files

File: Documentation/Wishlist.aux [hunk 3]
```diff
@@ -1,7 +0,0 @@
-\relax
```
Comment (ibell) [inline Documentation/Wishlist.aux]: Don't commit aux files

File: Documentation/Condenser.aux (hunk 2 above)
Comment (bo3mrh) [inline Documentation/Condenser.aux]: Sorry, I will try to avoid this next time.

File: Documentation/Wishlist.aux (hunk 3 above)
Comment (bo3mrh) [inline Documentation/Wishlist.aux]: Sorry, I will try to avoid this next time.

**Pull request merged**